docs/source/reference/api-server/api-server-admin-deploy.rst

```diff
@@ -48,11 +48,33 @@ Install the SkyPilot Helm chart with the following command:
     WEB_USERNAME=skypilot
     WEB_PASSWORD=yourpassword
     AUTH_STRING=$(htpasswd -nb $WEB_USERNAME $WEB_PASSWORD)
-    # Deploy the API server
-    helm upgrade --install $RELEASE_NAME skypilot/skypilot-nightly --devel \
-      --namespace $NAMESPACE \
-      --create-namespace \
-      --set ingress.authCredentials=$AUTH_STRING
+
+.. tab-set::
+
+    .. tab-item:: Basic Auth in API Server (Supports RBAC)
+        :sync: api-server-tab
+
+        Deploy the API server with basic auth enabled in the API server (supports RBAC):
+
+        .. code-block:: bash
+
+            helm upgrade --install $RELEASE_NAME skypilot/skypilot-nightly --devel \
+            --namespace $NAMESPACE \
+            --create-namespace \
+            --set apiService.enableUserManagement=true \
+            --set apiService.initialBasicAuthCredentials=$AUTH_STRING
+
+    .. tab-item:: Basic Auth in Ingress (No RBAC)
+        :sync: ingress-tab
+
+        Deploy the API server with basic auth enabled in the ingress controller (does not support RBAC):
```

Comment:
We should explain what this implies. The RBAC version seeems will cause the original way of distributing the API server URL `singleusername:singlepassword@api.server.url` won't have multi-tenancy?

How about we keep the RBAC an advanced way instead?

Reply:
Updated